ouroboros-network/ouroboros-network.cabal

```diff
@@ -306,6 +312,8 @@ test-suite test-cddl
   ghc-options:         -Wall
                        -Wno-unticked-promoted-constructors
                        -fno-ignore-asserts
+  if flag(nix)
+    cpp-options:       -DNIX
```

Comment:
Isn't this define a bit too generic? I think that this define is passed on when compiling all dependencies too.

Reply:
Then it's better to change it. I'll use `OUROBOROS_NETWORK_TEST_WITH_NIX`.

Reply:
Or even better, I'll drop `filepath` and `directory` dependencies to this test and just check paths, this way no `CPP` is needed.